applications/ecallmgr/src/ecallmgr_fs_notify.erl

```diff
@@ -256,7 +256,15 @@ register_overwrite(JObj, Props) ->
                  ]).
 
 -spec ensure_contact_user(ne_binary(), ne_binary(), ne_binary()) -> api_binary().
-ensure_contact_user(Contact, Username, Realm) ->
+ensure_contact_user(OriginalContact, Username, Realm) ->
+    Contact = case binary:split(OriginalContact, <<";">>, ['global']) of
+                  [<<>> | L] ->
+                      lists:foldl(fun(A,B) ->
+                                          <<B/binary, ";", A/binary>>
+                                  end, <<"sip:", Username/binary, "@", Realm/binary>>, L);
```

Comment:
If you're trying to just remove the `$;` prefix from the binary, why not just pattern-matching it?
You're deconstructing `OriginalContact` based on `;` then re-assembling it again, right?